coalib/bears/Bear.py

```diff
@@ -290,18 +293,26 @@ def __init__(self,
         :param message_queue: The queue object for messages. Can be ``None``.
         :param timeout:       The time the bear is allowed to run. To set no
                               time limit, use 0.
+        :param profile:       Enable profiling on this bear. Contains directory
+                              name or directory path where profiled files will
+                              be dumped, if ``True`` is provided files will be
+                              saved to the current directory.
         :raises TypeError:    Raised when ``message_queue`` is no queue.
         :raises RuntimeError: Raised when bear requirements are not fulfilled.
         """
         Printer.__init__(self)
 
         if message_queue is not None and not hasattr(message_queue, 'put'):
             raise TypeError('message_queue has to be a Queue or None.')
+        if profile and debugger:
+            raise ValueError("Can't run debugger and profiler at the same "
```

Comment:
avoid "Can't", and unnecessary use of `'`  Use simpler English.

and start long strings on the next line, so you dont need to break the string across lines

Reply:
Any suggestions here?